meson.options

```diff
@@ -20,6 +20,8 @@ option('enable_stress', type: 'boolean', value: 'true',
        description: 'Enable stress tests')
 option('kernel', type: 'string', value: 'vmlinuz',
        description: 'kernel image used to test schedulers')
+option('scheduler', type: 'string', value: '',
```

Comment:
Not sure if this is the right approach, but it's the only easy way I could think of getting this to work. My thought is if the github workflow is setup correctly it could use this variable to only build the configured scheduler.

Reply:
Would it make sense to indicate that it's for running veristat in the option name?

Reply:
Something like `veristat-scheduler`? I guess if it gets made more generic in the future it can be renamed.